### PR TITLE
Adds support for prioritizing processing.

### LINKED ISSUE #164: Specify preferred barcode ordering for download from Google
We will need to specify a preferred ordering of processing of barcodes (i.e. the order in which we request processing and download from Google).  This will allow us to accession books in an order of maximal stakeholder value.  Work will need to be done separately to determine what the order should be (e.g. public domain first, certain call number ranges first), but the google books app should have the ability to use a lookup by barcode to determine a priority sort ordering.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #164

## Why was this change made?
To provide a mechanism for prioritizing our requests for processing.

## Was the documentation (README, DevOpsDocs, wiki) updated?
Yes.

### CODE REVIEW

File: app/services/fetcher_service.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@ def populate_books
   # This is limited to the number of books specified in Settings.dor_services.marcxml_lookup.limit.
   # In addition, there is a wait of Settings.dor_services.marcxml_lookup.wait between each request.
   def fetch_marcxmls
-    Book.where(status: Book::READY).order(:barcode)
-        .limit(Settings.dor_services.marcxml_lookup.limit).find_each do |book|
+    Book.where(status: Book::READY).order(priority: :desc, barcode: :asc)
+        .take(Settings.dor_services.marcxml_lookup.limit).each do |book|
```
Comment (mjgiarlo) [inline app/services/fetcher_service.rb]: The change from `limit.find_each` to `take.each` is OK but I am curious about the reason for doing so.

Comment (justinlittman) [reply app/services/fetcher_service.rb]: For some reason (which I didn't track down), the ordering was ignored with find_each. Dunno?

Comment (mjgiarlo) [reply app/services/fetcher_service.rb]: Weird! No problem.

Comment (mjgiarlo) [reply app/services/fetcher_service.rb]: @justinlittman here we go: https://stackoverflow.com/a/19818566

**Pull request merged**